code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3759,3 +3759,77 @@ datum
 			alch_strength = 0.4
 			description = "A warm, late night drink, usually enjoyed during long winter nights."
 			reagent_state = LIQUID
+
+		fooddrink/alcoholic/grenadine
+			name = "Grenadine"
+			id = "grenadine"
+			fluid_r = 234
+			fluid_g = 19
+			fluid_b = 19
+			description = "A sticky, sweet and tart non-alcoholic bar syrup, used in cocktails for it's distinct bright red colour."
+			reagent_state = LIQUID
+
+		fooddrink/pinklemonade
```

Comment:
Could consider making this a child of lemonade. Doesn't *really* matter, just makes organizational sense, I think?

Reply:
sounds fair